web/templates/user_display_only_pill.hbs

```diff
@@ -3,7 +3,7 @@
         {{#if img_src}}
         <img class="pill-image" src="{{img_src}}" />
         {{/if}}
-        <span class="pill-label">
+        <span class="pill-label {{#if strikethrough}} strikethrough {{/if}}" >
```

Comment:
We name the property strikethrough here since the pill has no concept of a soft removal and it's just a visual decoration for the pill

Reply:
Yeah, this is definitely the way to do it.